DataStruturesPartI/MyStack.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataStruturesPartI;
+
+internal class MyStack<T>
```

Comment:
If you use internal here you should use internal within the related classes or use public